handleData.js

```diff
@@ -119,11 +127,52 @@ var Character = function(realName, gameName, level, gameClass, race) {
     this.characterLevel = level;
     this.characterClass = gameClass;
     this.characterRace = race;
+    this.inventory = [];
+    this.money = { "Copper Pieces"      : 0,
+                   "Silver Pieces"      : 0,
+                   "Gold Pieces"        : 0,
+                   "Platinum Pieces"    : 0,
+                   "Electrum Pieces"    : 0};
 };
 
 Character.prototype = Object.create(Player.prototype);
 Character.prototype.constructor = Character;
 
+Character.prototype.addItem = function(newItem) {
+    for (var item in this.inventory) {
+        if (this.inventory[item].name === newItem.name) {
+            this.inventory[item].quantity += newItem.quantity;
+            return "Added new quantity to existing quantity.";
+        }
+    }
+
+    this.inventory.push(newItem);
+    return "Added new item to inventory";
+}
+
+Character.prototype.removeItem = function(targetItem) {
+    for (var item in this.inventory) {
```

Comment:
I felt like I was being fancier by keeping the line short, but I honestly prefer the more verbose _for(blah; condition; blah)_ format